moveit_setup_assistant/launch/setup_assistant.launch

```diff
@@ -1,14 +0,0 @@
-<!-- Launch the MoveIt Setup Assistant, or just use rosrun -->
-<launch>
-
-  <!-- Debug Info -->
-  <arg name="debug" default="false" />
-  <arg unless="$(arg debug)" name="launch_prefix" value="" />
-  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
-
-  <!-- Run -->
-  <node pkg="moveit_setup_assistant" type="moveit_setup_assistant" name="moveit_setup_assistant"
-        launch-prefix="$(arg launch_prefix)"
-        output="screen"  required="true" />
-
-</launch>
```

Comment:
Why did you delete this file?  It is not part of your port.

Reply:
So that people don't try and run it.  I also thought that its such a simple launch file, we do not loss any useful information.

Reply:
you can't run a .launch file in ros2, also this file is not part of your port